src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationParsingTests.cs

```diff
@@ -7241,15 +7241,10 @@ public void TestOptParamMethodDeclarationWithNullValidation()
             }
         }
 
-        [Fact(Skip = "PROTOTYPE(BangBang)")]
+        [Fact]
         public void TestOptParamMethodDeclarationWithNullValidationNoSpaces()
         {
-            UsingStatement(@"void M(string name!=null) { }", options: TestOptions.RegularPreview, expectedErrors: new DiagnosticDescription[]
-            { 
-                    // (1,19): error CS8712: Space required between explanation-point and equals-sign here.
-                    // void M(string name!=null) { }
-                    Diagnostic(ErrorCode.ERR_NeedSpaceBetweenExclamationAndEquals, "!=").WithLocation(1, 19)
-            });
+            UsingStatement(@"void M(string name!!=null) { }", options: TestOptions.RegularPreview, expectedErrors: new DiagnosticDescription[0]);
```

Comment:
>new DiagnosticDescription[0] [](start = 115, length = 28)

[http://sourceroslyn.io/#Roslyn.Test.Utilities/Diagnostics/DiagnosticDescription.cs,694b729bf22fac05,references](http://sourceroslyn.io/#Roslyn.Test.Utilities/Diagnostics/DiagnosticDescription.cs,694b729bf22fac05,references) for these.

Reply:
Don't forget about this comment.

---
In reply to: [466677503](https://github.com/dotnet/roslyn/pull/46520#discussion_r466677503) [](ancestors = 466677503)

Reply:
Thanks, modified the tests to remove these extraneous bits.